Sources/PortalSwift/Portal.swift

```diff
@@ -344,7 +344,9 @@ public class Portal {
     let privateKeys = try await mpc.eject(
       method,
       withCipherText: withCipherText,
-      andOrganizationBackupShare: andOrganizationBackupShare
+      andOrganizationBackupShare: andOrganizationBackupShare,
+      andOrganizationSolanaBackupShare: nil,
```

Comment:
im confused why the changes are requiring the arguments to be passed as nil if they are optional anyways